docs/sources/get-started/component_controller.md

```diff
@@ -81,7 +81,7 @@ The overall health of a component is determined by combining the controller-repo
 An individual component's health is independent of the health of any other components it references.
 A component can be marked as healthy even if it references an exported field of an unhealthy component.
 
-## Handling evaluation failures
+## Handle evaluation failures
```

Comment:
Hmmm you made me read the topic section (instead of myopically focusing on just the linting warnings)  and... the title is better rewritten as `Reload the configuration file`